src/main/java/frc/robot/utils/MoPrefs.java

```diff
@@ -86,4 +109,72 @@ public static double getShooterPIDSetpoint() {
   public static double getShootFromWallHoodSetpoint() {
     return getDouble("Shoot From Wall Hood Setpoint", SHOOT_FROM_WALL_HOOD_SETPOINT);
   }
+
+  public static double getShooterKP() {
```

Comment:
Does anyone else find this code pattern a challenge to maintain?
Why do we need specific methods baked into a shared class to get values from a dictionary?
It has always seemed to me that this should be easier.  Specifically, MoPrefs should maintain the dictionary, but not know/care about what's in it.  The clients should be responsible for knowing their own keys.

Reply:
Well, MoPrefs is just a wrapper around ` edu.wpi.first.wpilibj.Preferences`. The only functionality it adds is the individual method calls. As far as I recall, these are the reasons for why we chose to create the MoPrefs instead of directly using the Preferences object:
1. To provide explicit default values for each preference
2. To make it very obvious what values are stored in the preferences
3. To prevent errors where typos in a string key ends up getting unexpected values

However, I do agree that this way of achieving that goal is pretty clunky.

Maybe a better way to do this would be to create a `enum MoPrefKeys` and refactor `MoPrefs` to only have one public method which takes a MoPrefKey and returns the value. Java's enums are basically just classes, so we could define a MoPrefKey to hold both the string key and an explicit default value.

That shouldn't be too hard to implement: I'll create a PR.

Reply:
See #27